Formula/crytic-compile.rb

```diff
@@ -0,0 +1,43 @@
+class CryticCompile < Formula
+  include Language::Python::Virtualenv
+
+  desc "Abstraction layer for smart contract build systems"
+  homepage "https://github.com/crytic/crytic-compile"
+  url "https://github.com/crytic/crytic-compile/archive/refs/tags/0.2.3.tar.gz"
+  sha256 "b41cc3ca6e43b70c2257dba5c1e794ca0881577f908acee9bc5bdfb132792e30"
+  license "AGPL-3.0-only"
+  head "https://github.com/crytic/crytic-compile.git", branch: "master"
+
+  depends_on "python@3.10"
+  depends_on "solc-select"
+
+  resource "pysha3" do
+    url "https://files.pythonhosted.org/packages/73/bf/978d424ac6c9076d73b8fdc8ab8ad46f98af0c34669d736b1d83c758afee/pysha3-1.0.2.tar.gz"
+    sha256 "fe988e73f2ce6d947220624f04d467faf05f1bbdbc64b0a201296bb3af92739e"
+  end
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    (testpath/"test.sol").write <<~EOS
+      pragma solidity ^0.8.0;
+      contract Test {
+        function f() public pure returns (bool) {
+          return false;
+        }
+      }
+    EOS
+
+    # crytic-compile is expected to invoke "solc" from solc-select
+    ENV.prepend_path "PATH", Formula["solc-select"].bin
```

Comment:
I don't think you need this. It won't work if you've done `brew unlink solc-select`, but we assume that's the case for the `test` block.

Reply:
Thanks for the review! Alright, if it's safe to assume that the dependencies will be linked, I'll drop it then :+1: